src/components/progress-circle/progress-circle.ts

```diff
@@ -124,13 +124,13 @@ export class MdProgressCircle implements OnDestroy {
   get mode() {
     return this._mode;
   }
-  set mode(m: ProgressCircleMode) {
-    if (m == 'indeterminate') {
+  set mode(_mode: ProgressCircleMode) {
```

Comment:
The underscore is used for private/internal class members. Function arguments should not use it.